src/models/index.d.ts

```diff
@@ -0,0 +1,47 @@
+import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";
+
+
+
+
```

Comment:
remove empty lines

Reply:
everything which is lint related in src/models should be ignored because those files are auto-generated